examples/sample-app/README.md

```diff
@@ -163,26 +169,17 @@ This section covers how to perform all the steps of building, deploying, and upd
     be added to the docker-registry service list so that it's reachable from other places.
 
 
-6. Confirm the registry is accessible (you may need to run this more than once):
-
-        $ curl -H 'Accept: application/json' -v `oc get service docker-registry --template="{{ .spec.portalIP }}:{{ with index .spec.ports 0 }}{{ .port }}{{ end }}/v2/" --config=openshift.local.config/master/admin.kubeconfig`
-
-    You should see:
-
-        "< Docker-Distribution-Api-Version: registry/2.0"
-
-
-7. Bind a user names `test-admin` to the `view` role in the default namespace so you can observe progress in the web console
+6. Bind a user names `test-admin` to the `view` role in the default namespace so you can observe progress in the web console
```

Comment:
I think this is unnecessary now too.  later in the steps they create a non-default project and create their app there, so there's no need to grant them access to the default project.

Note that until that project is created, they won't be able to see anything in the web console if they login as test-admin.